runtime/src/in_mem_accounts_index.rs

```diff
@@ -264,6 +271,42 @@ impl<T: IndexValue> InMemAccountsIndex<T> {
         result
     }
 
+    pub(crate) fn flush(&self) {
+        let flushing = self.flushing_active.swap(true, Ordering::Relaxed);
+        if flushing {
+            // already flushing in another thread
+            return;
+        }
+
+        self.flush_internal();
+
+        self.flushing_active.store(false, Ordering::Relaxed);
+    }
+
+    fn flush_internal(&self) {
+        let was_dirty = self.bin_dirty.swap(false, Ordering::Relaxed);
+        if !was_dirty {
+            // wasn't dirty, no need to flush
+            return;
+        }
+
+        let map = self.map().read().unwrap();
+        for (_k, _v) in map.iter() {
+            /*
+            if v.dirty() {
+                // step 1: clear the dirty flag
+                // step 2: perform the update on disk based on the fields in the entry
+                // If a parallel operation dirties the item again - even while this flush is occurring,
+                //  the last thing the writer will do, after updating contents, is set_dirty(true)
+                //  That prevents dropping an item from cache before disk is updated to latest in mem.
+                v.set_dirty(false);
+
+                // soon, this will update disk from the in-mem contents
+            }
+            */
+        }
+    }
```

Comment:
I guess this is OK to comment out for now?

Reply:
yes, that code didn't/doesn't exist yet but will soon. Showing you the basics of what it does.